packages/datagateway-common/src/card/advancedFilter.component.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { createShallow } from '@material-ui/core/test-utils';
-import TitleIcon from '@material-ui/icons/Title';
+import SubjectIcon from '@mui/icons-material/Subject';
```

Comment:
Hi Luna, hope you don't mind me chiming in! I think I see the problem here and it's definitely not your fault.

Basically you've attempted to import the `SubjectIcon` using the Material UI v5 method. However, for the moment, we're still using Material UI v4, which is why all the @material-ui references in the code aren't what you'll see in the up-to-date documentation. Fortunately, the v4 docs are still available at https://v4.mui.com/components/material-icons/ which advise us to use

```
import SubjectIcon from '@material-ui/icons/Subject';
```

So if you change your references to look like the above, there should be no problems at all! For reference, we are in the middle of trying to upgrade to Material UI v5 but this won't be done for at least a month or two yet (waiting on Diamond release first)